draft-ietf-ipsecme-ikev2-multiple-ke-00.xml

```diff
@@ -142,16 +139,20 @@ after we submit a new draft to ikev2-beyond64k-limit. -->
 			distinct PQC algorithms.
             </t>
 
-            <t>
-			The secrets established from each key exchange are combined in a way
-			such that should the post-quantum secrets not be present, the derived
-			shared secret is equivalent to that of the standard IKEv2; on the
-			other hand, a post-quantum shared secret is obtained if both classical
-			and post-quantum key exchange data are present.  This extension also
-			applies to key exchanges in IKE Security Associations (SAs) for
-			Encapsulating Security Payload (ESP) <xref target="RFC4303"/> or
-			Authentication Header (AH) <xref target="RFC4302"/>, i.e. Child SAs,
-			in order to provide a stronger guarantee of forward security.</t>
+            <t>IKE peers perform multiple successive key exchanges to establish
+            an IKE Security Association (SA).  Each exchange produces a piece of secret and
+            these secrets are combined in a way such that:
+            <ol type="(%c)">
+            <li>the final shared secret is computed from all of the component key exchange
+            secret;</li>
+            <li>if the responder does not understand the additional key exchange introduced
+            in this specification, the peers can ignore these key exchanges 
+            and the resulting shared secret will be the same as that defined in
+            <xref target="RFC7296"/>; and</li>
```

Comment:
I think it depends on whether the initiator includes a proposal without Additional Key Exchange transforms. So, in general, if the initiator makes performing additional key exchanges optional (by including two proposals - with and without them), then the unsupporting responder will select the proposal without Additional Key Exchange transforms, and the result will be IKE SA with the keys computed according to RFC 7296. If the initiator doesn't provide such an option, IKE SA won't be ctreated.

I think that the main point of the original text was that we don't change key derivation in case additional key exchanges were not used. Whether they are used or not depends on both sides, not only on responder.

Reply:
How about if we say this instead?

```suggestion
            <li>the shared secret as specified in <xref target="RFC7296"/> is obtained
            unless both peers support and agree to use the additional key exchanges introduced
            in this specification; and</li>
```

Reply:
Perfect!